deployment/terraform/azure/container_app/examples/azure-integration/main.tf

```diff
@@ -0,0 +1,107 @@
+variable "port_client_id" {
+  type = string
+}
+variable "port_client_secret" {
+  type = string
+}
+variable "port_base_url" {
+  type = string
+  default = ""
+}
+
+module "ocean_integration" {
+  source       = "../.."
+
+  # required port parameters so that the integration could communicate with Port
+  port = {
+    client_id     = var.port_client_id
+    client_secret = var.port_client_secret
+    base_url = var.port_base_url
+  }
+
+  initialize_port_resources = true
+
+  # required port integration parameters so Port could identify the integration
+  integration = {
+    type       = "azure"
+    identifier = "az1"
+    config     = {
+    }
+  }
+  # optional port integration parameters
+  subscription_id = "/subscriptions/xxxxxx"
+  location = "East US 2"
+
+  image = "ghcr.io/port-labs/port-ocean-azure:v0.1.0rc11"
```

Comment:
I think this should be a variable ( atleast the image tag )

Any parameter which might be dynamic should be a varaible. This will make it easier to use this example and implement it for customers

Reply:
fixed